python/torch_mlir/extras/fx_importer.py

```diff
@@ -1212,7 +1230,10 @@ def _import_symbolic_torch_op(
             ), f"Unsupported builtin function for symbolic types: {target} with args {node.args}"
             concrete_target = getattr(torch_op, op_overload)
         else:
-            concrete_target = SYMBOLIC_OP_TO_TORCH_OP.get((target, len(node.args)))
+            if torch.__version__.split("+")[0] <= "2.1.0":
```

Comment:
I think we should have these version checks at the module level bound to constants like `_IS_TORCH_2_1_OR_EARLIER`. saves some inline work and also makes the code more readable and clear for when we can drop branches.